src/Bridges/Constraint/bridges/det.jl

```diff
@@ -144,7 +144,7 @@ function MOI.supports_constraint(
 end
 
 function MOI.Bridges.added_constrained_variable_types(::Type{<:LogDetBridge})
-    return Tuple{Type}[]
+    return Tuple{Type}[(MOI.Reals,)]
```

Comment:
Were any bridge already returning `MOI.Reals` here ?

Reply:
Nope. I think it was just an oversight.